### ISSUE TITLE
When I try to install, I get the error "Could not find a version that satisfies the requirement pysce"

### ISSUE THREAD

**Issue by wcyoung08** [jkitchin/pycse#27]
Hi,

I'm on a Windows 10 machine trying to download pysce, but when I run 'pip install pysce' from the ipython console, I get the error:

`Could not find a version that satisfies the requirement pysce (from versions: )
No matching distribution found for pysce`

I'm a bit new to github and python in general, so hopefully the solution is simple and I just don't know enough yet to figure it out on my own.

**Comment by wcyoung08** [comment]
So I really hope that wasn't my issue because I spent an entire morning working on this, and ended up writing a different form of code to solve my problem I was working on. 

I want to mention for anyone that might read this thread later that I was using python(x,y), which I realized couldn't actually run past python 2. I uninstalled that and got the anaconda console with python 3.7, and when I ran it again, it worked. I want to think that was what solved the problem and not the fact that I spelled it wrong, but either way it's fixed. Thank you!
